docs/user-guides/4-multiplayer.md

```diff
@@ -0,0 +1,12 @@
+---
+id: multiplayer
+title: Using Multiplayer
+---
+
+The Webaverse has multiplayer support. This is in the form of individual XRPackages that implement multiplayer however they want, using some information that the XRPackage system provides to them.
+
+If you want to use multiplayer in your world, you'll need to import a Hubs or multiplayer package. An example of this is <a href="https://github.com/webaverse/multiplayer" target="_blank" rel="noopener noreferrer">the Webaverse `multiplayer`</a> package.
```

Comment:
This is kind of misleading

 We can do whatever multiplayer, so nothing is "needed". What is mentioned is just some examples.

Reply:
Thanks, reworded this a bit!